src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetInstaller.cs

```
@@ -40,8 +40,31 @@ public Task<bool> CanInstallAsync(InstallRequest installationRequest)
             return Task.FromResult(!string.IsNullOrWhiteSpace(installationRequest.Identifier) && !Directory.Exists(installationRequest.Identifier));
         }
 
+        public IManagedTemplatesSource Deserialize(IManagedTemplatesSourcesProvider provider, string mountPointUri, object details)
+        {
+            return new NuGetManagedTemplatesSource(this, mountPointUri, details as Dictionary<string, string>);
```

Aren't we hiding the error here?
```csharp
details as Dictionary<string, string>
```
Maybe we should throw?